backend/utils/graph_db_utils.py

```diff
@@ -23,14 +21,14 @@ def test_connection():
     finally:
         driver.close()
     
-def create_goal(name, description):
+def create_goal(goal_name, goal_description):
     try:
         session = driver.session()
         query = """
-        MERGE (g:Goal {name: $name, description: $description})
+        MERGE (g:Goal {goal_name: $goal_name, goal_description: $goal_description})
         RETURN g
         """
-        result = session.run(query, name=name, description=description)
+        result = session.run(query, goal_name=goal_name, goal_description=goal_description)
```

Comment:
I haven't tried it so correct me if I'm wrong but you may be able to pass the arguments without the `=` operator as they have the same name as the parameters
```suggestion
        result = session.run(query, goal_name, goal_description)
```

Reply:
Does not work, also discussed on call